src/js/queries/documentPageFragment.js

```diff
@@ -21,6 +21,7 @@ const documentPageFragment = `
           document {
             fileSize
             filename
+            url
           }
         }
       }
```

Comment:
same here, is this query being used anywhere?

Reply:
I deleted it on my branch, because i thought it was leftover, but ill put it back if its coming into play

Reply:
I kept it because I wasn't sure if your branch would use it. I can delete those 2 files from this branch too.